app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -1,13 +1,10 @@
-/* Placeholder for backend dummy app */
 (function($) {
   SpreePayuIntegration = {
-    updateSaveAndContinueVisibility: function(e) {
+    updatePayuSelectedClass: function(e) {
```

Comment:
1. there is an initial call to this method that adds/removes class based on which payment method is selected on page load

2. there can be more than 2 payment methods, if user would switch between two other. non-payu methods, it will toggle the class but it should not, until payu is choosen